components/playback/controls/player/PopInput.vue

```diff
@@ -0,0 +1,231 @@
+              border: 0px;
+            "
+          ></b-form-input>
+        </span>
+      </div>
+      <div
+        class="div__popinput-controls-content-input-feedback"
+        :style="
+          'width: ' + input_width + 'px;' + 'top:' + input_feedback_top + 'px;'
+        "
+      >
+        {{ invalid_text }}
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+import Vue from "vue";
+import uuid from "@tofandel/uuid-base62";
+import BootstrapVue from "bootstrap-vue";
+import { BButton } from "bootstrap-vue";
+import { BFormInput } from "bootstrap-vue";
+Vue.use(BootstrapVue);
+Vue.component("BFormInput", BFormInput);
+Vue.component("BButton", BButton);
+import "bootstrap/dist/css/bootstrap.min.css";
+Vue.use(uuid);
+
+export default {
+  name: "PopInput",
+  props: {
+    title_label: { type: String, default: "", required: false }, // title_text (str) (optional, defaults to empty string "")
+    key_placeholder: { type: String, default: "" }, // placeholder (str)
+    invalid_text: { type: String, default: "" }, // invalid_text (str)
+    input_check: { type: Boolean, default: true, required: false }, // spellcheck (optional bool=True)
+    user_key: { type: String, default: "", required: false }, // field_value (str) (optional, defaults to empty string "")
+    input_width: { type: Number, default: 0 }, // textbox_width (int)  [pixels]
+    block: { type: Boolean, default: false, required: false }, // disabled (optional bool=False) (not able to type into input)
+  },
+  data() {
+    return {
+      input_value_key: this.user_key,
+      input_width_background: this.input_width + 4, // This is required as the red/green boxes around the input widget requirement based on feedback introduced the need its not in Mockflow
+      // very essential else the input box would appear poping out on the right side outside the background, request to consult Eli or Raghu
+    };
+  },
+  computed: {
+    input_key_action: function () {
+      // This is a very sensitive computed function as its invoked on every key entry by user action
+      // the function would never have any processing its only responsible to pass the value of string to the parent component
+      // any modification to add logic might impact depedent functionalities, request to consult Eli or Raghu
+      this.$emit("update:user_key", this.input_value_key);
+      return this.invalid_text === "";
+    },
+    input_height_background: function () {
+      return this.title_label !== "" ? 100 : 60;
+    },
+    input_widget_top: function () {
+      return this.title_label !== "" ? 40 : 0;
+    },
+    input_feedback_top: function () {
+      return this.title_label !== "" ? 88 : 48;
+    },
+  },
+  methods: {},
+};
+</script>
+<style type="text/css">
+.div__popinput-background {
+  transform: rotate(0deg);
+  box-sizing: border-box;
+  padding: 0px;
+  margin: 0px;
+  background: rgb(17, 17, 17);
+  position: absolute;
+  /* height: 100px; */
```

Comment:
remove old commented code when submitting pull requests, unless there is a specific reason to leave it in